pkg/hubble/relay/observer.go

```diff
@@ -31,6 +33,21 @@ import (
 // ensure that Server implements the observer.ObserverServer interface.
 var _ observerpb.ObserverServer = (*Server)(nil)
 
+func relayStatusResponse(numPeers int, failedPeers []string) *observerpb.GetFlowsResponse {
+	return &observerpb.GetFlowsResponse{
+		Time:     ptypes.TimestampNow(),
+		NodeName: node.GetName(),
```

Comment:
Does the node name makes sense in the context of hubble-relay? Well, I guess we have to fill this anyway and I see no better alternative.